contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
+        const allAnswerPaths = []
+        const allAnswers = []
+        const answersUsed = []
+
+        // Log ids of all answers in section
+        this.questions.forEach((question) => {
+            question.answers.forEach((answer) => {
+                allAnswers.push(answer.id)
+            })
+        })
+
+        // Log ids of answers used in minimum paths for recommendations
+        const allRecommendationPaths = this.minimumPathsForRecommendations;
+        for (const maturity in allRecommendationPaths) {
+            allRecommendationPaths[maturity].forEach((question) => {
+                if (!answersUsed.includes(question.answer.id)) {
+                    answersUsed.push(question.answer.id)
+                }
+            })
+        }
+
+        // Create paths until all answers are used
+        while (answersUsed.length !== allAnswers.length) {
+            const neededNextQuestions = [];
```

Comment:
Minor optimisation I think you can make here - `neededNextQuestions` could go from a list to `neededNextQuestion` which is either an id or null. When `neededNextQuestion` is truthy, you can then have
```js
const validAnswers = currentQuestion.answers.filter(answer => answer.nextQuestion && answer.nextQuestion.sys.id == neededNextQuestion)
```
instead of looking up the first element
and instead of unshifting you can just set `neededNextQuestion` to whatever the new id is

Reply:
I get you, I went back and forth with this and probably could have gone back again. I think I did it like this with a view to making sure that needed questions later on didn't get dumped if interim questions weren't technically needed eg you have an unused answer for Q6 so unshift that, you select a non-needed Q5 answer to get there and don't unshift that one, then you have an unused Q4 answer that skips Q5 but goes to Q6... So the idea behind the array being that you keep the ones you need and if you have a 'skipper' you could check that it's not skipping anything necessary. 
But I ended up not making it that clever in the end, so the string version would work as it's written!
Do you think I should do the string thing or try to do the clever version?